lib/patches/rest_client_patch.rb

```diff
@@ -0,0 +1,29 @@
+require 'restclient/version'
+if RestClient::VERSION >= "2.0.0" && RestClient::VERSION <= "2.1.0"
+  require 'restclient/request'
+  RestClient::Request.module_eval do
```

Comment:
Note, this autoloads rest client regardless if you're going to use it.  I'm open to suggestions on how to avoid this.

Reply:
I'm trying to remember why Matthew convinced us a few years ago to use the module_eval thing instead of prepend.  Either way, I think I'd be able to remove the require above it but both approaches would still autoload the constant for all processes.

Reply:
Yeah, nothing we can do about requiring it all the time.

Reply:
This will monkey patch the module itself. A `prepend` will override the parent of this class.
Pretty sure we want don't want `prepend` here.

reading the comments, I did not realize that we are monkey patching `rest-client` code onto `rest-client` because they haven't pulled the trigger in July 2017

Reply:
yeah, good point @kbrock regarding monkey patch vs. prepend.  I don't think it matters here but `module_eval` seems better for replacing a broken method vs. getting in "front" of it.

Yeah, rest-client is mostly dead.  I've reached out to see if we can help maintain it but have heard nothing.  Perhaps, we should fork it permanently at some point in the future.   I don't know enough of it to be comfortable making changes throughout the codebase.

Reply:
I'll verify I can remove the require line too.

Reply:
I can't remove the require line since it doesn't autoload rest-client's request.